### DISCUSSION TITLE
OK to distribute "param.get.pd" in a GPL-3.0 project?

### DISCUSSION THREAD

**Discussion by GModal** [pierreguillot/Camomile#266] [Q&A] [answered]
Hi Pierre,

I made a LibPd project that runs Pd patches as a headless JACK client. I've been using the param.get object to transfer parameters from the cmd line to the patch.

Is it OK to include your abstraction (the project is GPL-3.0)? I could create a receive obj with a different name, etc., but I like your "standard."

Thanks!
Doug

**Comment by pierreguillot** [comment, marked as answer]
Yes, of course, you can use all the abstractions :)

**Reply by GModal** [reply]
Thanks -- I appreciate it! I just made it public.

It's still alpha : https://github.com/GModal/pd2jack
